src/MSBuild/XMake.cs

```diff
@@ -1275,7 +1272,7 @@ private static void ResetGatheringSwitchesState()
                     }
                     else
                     {
-                        Evaluation.Project project = projectCollection.LoadProject(projectFile, globalProperties, toolsVersion);
+                        Project project = projectCollection.LoadProject(projectFile, globalProperties, toolsVersion);
```

Comment:
Isn't this ambiguous between the fancy logger project and evaluation project without the tag?

Reply:
Jk